types/stream-chat/index.d.ts

```diff
@@ -113,6 +113,9 @@ export interface Reaction {
   [propName: string]: any;
 }
 
+export interface ReactionRequest {
+  type: string;
```

Comment:
reactions have more fields, @Bogdan-D please help here :)

Reply:
This is just for sending a reaction though. I was working on the typescript stuff and it was failing because we reused the Reaction response object for this.

Reply:
@jaapbakker88  Which properties of `Reaction` were breaking? I can see `message_id`, `user` as required properties on `Reaction` type. They are used for request using server side token. So maybe we can make them optional!!

Reply:
@vishalnarkhede updated to use only the `type` as a required property. It was indeed breaking on the rest.